### PR TITLE
Hud and Screen related changes

### PR DESCRIPTION
**Pull request by shedaniel**
The most noticeable change is the `draw` to `render` one. It makes sense to change because some already had been using `render` and all screens are using `render`.

### CODE REVIEW

File: mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping [hunk 1]
```diff
@@ -118,7 +118,7 @@ CLASS cwq net/minecraft/client/gui/widget/EntryListWidget
 		ARG 1 renderSelection
 	METHOD setScrollAmount (D)V
 		ARG 1 amount
-	METHOD setSelected (Lcwq$a;)V
+	METHOD setSelected selectEntry (Lcwq$a;)V
```
Comment (modmuss50) [inline mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping:121]: This should not have a name assgined to it.

**Review by modmuss50 — CHANGES_REQUESTED** [review]
One small issue

**Comment by modmuss50** [comment]
Ive merged + fixed that issue manually due to changes. Thanks

File: mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping (hunk 1 above)
Comment (shedaniel) [reply mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping:121]: Oops sorry for that

**Pull request merged**